.github/workflows/vercel-preview.yaml

```
@@ -1,25 +1,39 @@
-  group: ${{ github.ref }}-vercel-preview
+  group: ${{ format('{0}-vercel-preview', github.ref) }}
   cancel-in-progress: true
-
 jobs:
-  Deploy-Preview:
+  Deploy-Preview-Push:
+    if: github.event_name == 'push'
     runs-on: ubuntu-latest
     permissions:
-      id-token: write
+    permissions:
```

actionlint (1.7.7)

key "permissions" is duplicated in "Deploy-Preview-Push" job. previously defined at line:29,col:5

(syntax-check)